### PR TITLE
Updated source controller install

### PR DESCRIPTION
**Pull request by odinnordico**
Which other branches should this be merged with (if any)?
Only TAP1.1

### CODE REVIEW

File: source-controller/install-source-controller.md [hunk 1]
```diff
@@ -30,7 +30,10 @@ To install Source Controller:
     $ tanzu package available list controller.source.apps.tanzu.vmware.com --namespace tap-install
     - Retrieving package versions for controller.source.apps.tanzu.vmware.com...
       NAME                                     VERSION  RELEASED-AT
-      controller.source.apps.tanzu.vmware.com  0.2.0    2021-09-16T00:00:00Z
+      controller.source.apps.tanzu.vmware.com  0.2.0    2021-11-17 19:00:00 -0500 -05
+      controller.source.apps.tanzu.vmware.com  0.2.1    2022-02-09 19:00:00 -0500 -05
```
Comment (rashedkvm) [inline source-controller/install-source-controller.md:34]: Remove those versions - those don't support optional values schema 
```
      controller.source.apps.tanzu.vmware.com  0.2.0    2021-11-17 19:00:00 -0500 -05
      controller.source.apps.tanzu.vmware.com  0.2.1    2022-02-09 19:00:00 -0500 -05
```

Comment (odinnordico) [reply source-controller/install-source-controller.md]: Removed

**Comment by rashedkvm** [comment]
Should there be an example to show how to install the Source Controller with a values flag? 

@heyjcollins any thoughts?

```sh
tanzu package install source-controller -p controller.source.apps.tanzu.vmware.com -v 0.3.2 -n tap-install -f ca-cert.yaml
```

**Comment by RichardJJG** [comment]
If this is still a work in progress, please convert it to a draft and then convert it back again when finished. We don't want to merge this prematurely.

**Review by rashedkvm — APPROVED** [review]

**Comment by heyjcollins** [comment]
> Should there be an example to show how to install the Source Controller with a values flag?
> 
> @heyjcollins any thoughts?
> 
> ```shell
> tanzu package install source-controller -p controller.source.apps.tanzu.vmware.com -v 0.3.2 -n tap-install -f ca-cert.yaml
> ```

@rashedkvm - yes we definitely need to add instructions for how to install Source Controller with a values file.

**Review by rashedkvm — APPROVED** [review]

**Comment by HenryBorys** [comment]
@heyjcollins Is this ready for merging? Please let me know.

**Comment by rashedkvm** [comment]
> @heyjcollins Is this ready for merging? Please let me know.

@HenryBorys I have reviewed this with Josh. We are ready to merge this doc change. Thank you!

**Pull request merged**